common/config/presets/tari_config_example.toml

```diff
@@ -178,6 +178,10 @@ scan_for_utxo_interval = 180
 base_node_service_refresh_interval = 30
 # The maximum age of service requests in seconds, requests older than this are discarded
 base_node_service_request_max_age = 180
+# The cooldown period between balance enquiry checks in seconds; requests faster than this will be ignored.
+# For specialized wallets processing many batch transactions this setting could be increased to 60 s to retain
+# responsiveness of the wallet with slightly delayed balance updates (default: 1)
+#balance_enquiry_cooldown_period = 1
```

Comment:
I honestly still don't think this belongs here. It is soooooo niche as to be noise. @mikethetike thoughts?

Reply:
Removed from the default config as proposed